formats/financial_release/publisher/details.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "title",
+     "sub_title",
+     "content",
+     "published_date"
+   ],
+  "properties": {
+    "title": {
+      "type": "string"
+    },
+    "sub_title": {
```

Comment:
Does this actually fit into the `description` field that lives at the top level of the document? For context, if I recall correctly, that is also displayed underneath search results.